tests/test_parser.py

```diff
@@ -3,12 +3,10 @@
 
 from ruuvitag_ble import RuuvitagBluetoothDeviceData
 
-OUTDOOR_SENSOR_DATA = (
+V5_SENSOR_DATA = (
     b"\x05\x05\xa0`\xa0\xc8\x9a\xfd4\x02\x8c\xff\x00cvriv\xde\xad{?\xef\xaf"
 )
-INDOOR_SENSOR_DATA = (
-    b"\x05\x0e\xa4M~\xc8\x18\xfc\xbc\xfd\xf0\xff\xb4+\xf6\x00\x10<\xd97\x0f\xf7\xaa\x48"
-)
```

Comment:
Please keep this here – maybe rename it to `V5_INDOOR_SENSOR_DATA` if need be.

Reply:
It is completely unused is it not?  I can certainly keep it if needed but would suggest commenting out perhaps?

Reply:
It's unused by tests, yeah - the idea was to keep it around for verification that a parser would sensibly parse some outdoor-y data. Either way, removing it is out of scope for a PR that would add support for a new data format :)

Reply:
Sure agreed, I commented it out, is that sufficient?

Reply:
Yep, let's go with that! :)